Lib/game.cpp

```diff
@@ -66,8 +66,7 @@ class Game {
 
     const char* error = 0;
     enemy_sprite_ = playdate_->graphics->loadBitmap("data/nazi.png", &error);
-    if (error)
-      playdate_->system->logToConsole("error: %s", error);
+    if (error) playdate_->system->logToConsole("error: %s", error);
```

Comment:
Maybe add {} here?

Reply:
We need to update clang-format config for it

Reply:
I think that clang-format can't do it, I think that it doesn't add new tokens but rearranges the existing. I think that clang-tidy can do that.